### PR TITLE
Bug bounty: CSRF on RD Refresh Opps Page

### PR DESCRIPTION
**Pull request by force2b**
# Critical Changes

- The Refresh Opportunities button behavior in Recurring Donations has been modified to prompt the user to click Continue before the actual process of updating the RD's opportunities starts.

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by force2b** [comment]
**lurch: attach W-039489

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HsLOQA0" target="_blank">W-039489</a>

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -2122,6 +2122,15 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>The total Amount of all Closed Won Opps is not equal to the RD Amount.</shortDescription>
         <value>The combined Amount of all related Closed Won and Pledged Opportunities must match the Amount of a Fixed Length Recurring Donation.</value>
     </labels>
+    <labels>
+        <fullName>RD_RefreshOppsContinueMessage</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Confirmation message displayed to user before the process runs</shortDescription>
+        <value>This process updates all future installment Opportunities to match the current data for this Recurring Donation. For some installments, NPSP may delete and recreate new installments.
```
Comment (force2b) [inline src/labels/CustomLabels.labels]: @sam-knox Here is the text noted in the WI chatter field.

File: src/components/UTIL_PageHeader.component [hunk 2]
```diff
@@ -63,6 +63,18 @@
...
         description="Whether the save and new button should be disabled, defaults to false."/>
 
+    <!-- Continue attributes -->
+    <apex:attribute name="showcontinueBtn" type="Boolean" default="false"
+                    description="Set to false to hide continue button, default is false"/>
+    <apex:attribute name="continueAction" type="ApexPages.Action" default=""
+                    description="Action for continue button."/>
+    <apex:attribute name="continueLabel" type="String"
+                    description="Label for continue button. When blank will display btnContinue label."/>
+    <apex:attribute name="continueImmediate" type="Boolean" default="false"
```
Comment (npsp-reedestockton) [inline src/components/UTIL_PageHeader.component]: description does not match default value.

Comment (force2b) [reply src/components/UTIL_PageHeader.component]: Very good catch. So much for copy/paste. I also fixed other descriptions that were wrong for the same reason.

**Comment by rponti-sforg** [comment]
QE testing for this PR and associated WI is now complete.

For further testing details, see:
https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000001HsLOQA0/view
https://salesforce.quip.com/aap8AdpJIXEI

Currently monitoring 'rebuild' of **Feature Test - Namespaced**.  Once test successfully completes, will merge PR.  Once post-merge builds have completed and are marked as 'success', will set corresponding WI to Closed.

**Pull request merged**